tests/tensorflow/helpers.py

```diff
@@ -25,6 +26,11 @@
 from examples.tensorflow.common.object_detection.datasets.builder import COCODatasetBuilder
 from examples.tensorflow.classification.datasets.builder import DatasetBuilder
 
+if version.parse(tf.__version__) < version.parse('2.6'):
+    from tensorflow.python.keras.applications import imagenet_utils
+else:
+    from keras.applications import imagenet_utils
+
```

Comment:
Those lines can be moved to `nncf/tensorflow/tf_internals.py` for clean code.

Reply:
That's right. Updated dependencies!